code/game/machinery/computer/communications.dm

```diff
@@ -194,6 +194,8 @@
 			var/datum/map_template/shuttle/shuttle = locate(params["shuttle"]) in shuttles
 			if (!istype(shuttle))
 				return
+			if (!can_purchase_this_shuttle(shuttle))
```

Comment:
this functions name stands out, may be best as just can_purchase()

Reply:
I took the proc from tg, in no small part to help with future compatibility - keeping the name the same will mean less trouble down the line porting other communications console things from tg.

Reply:
Very well then 👌